src/Arcus.Messaging.Tests.Integration/appsettings.json

```diff
@@ -6,12 +6,18 @@
     "Infra": {
```

Comment:
I presume we've added all the required variables?

Reply:
In Azure DevOps? Yea, otherwise the tests wouldn't be successful. 😄